coda.io.website.json

```diff
@@ -0,0 +1,31 @@
+{  
+    "providerId":"coda.io",
+    "providerName":"Coda",
+    "serviceId":"website",
+    "serviceName":"Coda Site",
+    "version": 1,
+    "syncPubKeyDomain" : "coda.io",
+    "logoUrl":"https://domainconnect.org/wp-content/uploads/2016/06/wix-300x103.png",
+    "description":"Enables a custom domain to work with Coda",
+    "variableDescription":"`host` refers to the subdomain that the user wishes to use (not just limited to 'www'). `subdomain` refers to the subdomain for Coda's custom domain server. `docId` refers to the coda document used for custom domain. `verification` refers to the TXT host for doc ownership verification, differs based on A or CNAME. `ipAddress` refers to the proper IP for an apex domain to point to.",
+    "records":[  
+       {  
+          "type":"CNAME",
+          "host":"%host%",
+          "pointsTo":"%subdomain%.coda.io",
+          "ttl":3600
+       },
+       {
+           "type": "TXT",
+           "host": "%verification%",
+           "data": "Doc:%docId%",
```

Comment:
It'd be more future-proof to make the verification data opaque here with no mention of docs or doc IDs.

Seems annoying to change or update an existing domain connect template, but in the future we may want to allow domain owners to assert that they're willing to serve any document owned by a particular Coda user ID or any document belonging to a particular Coda workspace

Reply:
Sounds like the changing is done via version updating? I know that Domain Connect folks prefer as much static data as possible, and to only really use variables whenever data is not static:

<img width="1025" alt="Screen Shot 2022-12-30 at 5 10 40 PM" src="https://user-images.githubusercontent.com/8244301/210114876-fb647bbb-dede-4f4e-946c-4a022077f1bc.png">

https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#variables-and-security

Reply:
I think `host` is supposed to be `_coda.%host%`, which seems like what the linked documentation is talking about

Personally I don't see any added security in forcing `data` TXT contents begin with `Doc`: it won't prevent any hijacking (people can validate the wrong doc while still using a `Doc:` prefix) vs having an opaque blob but it'd makes it harder for us to add new features in the future

Reply:
@dweitzman-codaio the advisory in the specification to keep the variable data portion as small as possible is mostly related to the templates not using signing, where such template might be used as phishing vector to pass for example DNS validation for SSL certificate issue. With signing enabled the risk is obviously lower but it still should be considered in the template creation.